src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -793,10 +793,13 @@ public sealed partial class ProgIdAttribute : System.Attribute
     }
     public static partial class RuntimeEnvironment
     {
+        [System.ObsoleteAttribute("This property is no longer supported.")]
```

Comment:
Why in some cases do we name the member (e.g. "GetRuntimeInterfaceAsIntPtr(Guid, Guid) is no longer supported.") and in some cases we don't (e.g. "This property is no longer supported.")?

Reply:
Because I was being sloppy it looks like - there is no good reason. I would guess this is because I did the first one because it was a property and then copied an example from another method that had the name in it and updated it. I will submit an update.

Reply:
See https://github.com/dotnet/runtime/pull/50693.

Reply:
:-)